src/index.js

```diff
@@ -27,6 +27,11 @@ class ScrollableList extends Component {
 
   componentWillReceiveProps(props) {
     if (this.props.data !== props.data) {
+
+      if (props.data == null) {
+        props.data = [];
+      }
+
       this.setState({
         dataSource: this.ds.cloneWithRows(props.data),
```

Comment:
Maybe this is a better solution (skips one `if` check and doesn't mutate data by hand): `dataSource: this.ds.cloneWithRows(props.data || []),`